pyerrors/obs.py

```diff
@@ -1822,14 +1827,51 @@ def covobs_to_obs(co):
         means = [means]
 
     for i in range(len(means)):
-        ol.append(covobs_to_obs(Covobs(means[i], cov, name, pos=i, grad=grad)))
+        ol.append(covobs_to_obs(Covobs(float(means[i]), cov, name, pos=i, grad=grad)))
     if ol[0].covobs[name].N != len(means):
         raise Exception('You have to provide %d mean values!' % (ol[0].N))
     if len(ol) == 1:
         return ol[0]
     return ol
 
 
+class Meas(Obs):
+    """Class for a scalar measurement.
+
+    Convenience wrapper for scalar measurements.
+    """
+
+    def __init__(self, value: Union[float, int], dvalue: Union[float, int], name: Optional[str] = None):
+        """ Initialize Meas object.
+
+        Parameters
+        ----------
+        value : float
+            Mean value of the measurement.
+        dvalue : list or array
+            Standard error of the measurement.
+        name : Optional[str]
+            Optional name identifier for the measurement. If none is specified, a random uuid
+            string is used instead.
+        """
+        if not isinstance(value, (float, int)):
+            raise TypeError(f"value has to be a flaot or int, not {type(value)}")
+        if not isinstance(dvalue, (float, int)):
+            raise TypeError(f"dvalue has to be a float or int, not {type(dvalue)}")
+        super().__init__(samples=[], names=[], means=[])
+        if name is None:
+            name = uuid.uuid4().hex
```

Comment:
Hey @fjosw,
the class looks good to me. Thanks for the idea and addition. I just wanted to ask if this is something that we want. I do not mind it, and if there is a specific use case for you, we can, of course, leave it in.